.github/workflows/downstream-project-spl.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           source .github/scripts/downstream-project-spl-common.sh
 
           sudo apt update
-          sudo apt install libudev-dev binutils-dev libunwind-dev -y
+          sudo apt install libudev-dev binutils-dev libunwind-dev protobuf-compiler -y
```

Comment:
cc/ @yihau

kinda surprised we didn't have this dragged in already :thinking:

Reply:
yeah. this script is steal from https://github.com/solana-labs/solana-program-library/blob/master/ci/install-build-deps.sh

btw, the L99 may need your help as well 😞 
https://github.com/solana-labs/solana/blob/8970a3751a4de7ac7656af2e0921bc09b34e402e/.github/workflows/downstream-project-spl.yml#L99

Reply:
btw, I'm installing this one to docker image.

Reply:
uploaded and check1 just passed 🎉 

![Screenshot 2023-08-24 at 1 21 23 AM](https://github.com/solana-labs/solana/assets/8209234/64e00844-078c-4c50-95e7-326d6b91ddba)

Reply:
the ci passed on Buildkite side! I think we can ship this one when we have the same update to line 99 :trollface:

Reply:
ah missed that one!  should probably find a common home to specify system dependencies :thinking: